### PR TITLE
[Blocks] Disable link button when multiple blocks are selected

### PR DESCRIPTION
**Pull request by remidej**
### What does it do?

Disables the link button when multiple block nodes are selected
### Why is it needed?

Creating a link that spanned over multiple blocks created issues

### How to test it?

Select mulitple blocks, the link button should be disabled

Keep your selection within the same block, the link button should be enabled

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 1]
```diff
@@ -507,6 +507,31 @@ const LinkButton = ({ disabled }) => {
     return Boolean(match);
   };
 
+  const isLinkDisabled = () => {
+    // Always disable the whole editor is disabled
```
Comment (simotae14) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:511]:
```suggestion
    // Always disabled when the whole editor is disabled
```

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 2]
```diff
@@ -507,6 +507,31 @@ const LinkButton = ({ disabled }) => {
...
+    if (disabled) {
+      return true;
+    }
+
+    // Always enable when there's no selection
```
Comment (simotae14) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:516]:
```suggestion
    // Always enabled when there's no selection
```

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 3]
```diff
@@ -507,6 +507,31 @@ const LinkButton = ({ disabled }) => {
...
+    if (!editor.selection) {
+      return false;
+    }
+
+    // Get the block nodes closest to the anchor and focus
```
Comment (simotae14) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:521]:
```suggestion
    // Get the block node closest to the anchor and focus
```

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 4]
```diff
@@ -507,6 +507,31 @@ const LinkButton = ({ disabled }) => {
...
+    const anchorNodeEntry = Editor.above(editor, {
+      at: editor.selection.anchor,
+      match: (node) => node.type !== 'text',
+    });
+    const focusNodeEntry = Editor.above(editor, {
+      at: editor.selection.focus,
+      match: (node) => node.type !== 'text',
+    });
+
+    // Disable if the anchor and focus are not in the same block
```
Comment (simotae14) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:531]:
```suggestion
    // Disabled if the anchor and focus are not in the same block
```

**Comment by simotae14** [comment]
There is another bug, unrelated to yours, when you open the Edit view, there isn't editor selection and you click the link button the first click doesn't work

**Comment by remidej** [comment]
@simotae14 you're right, it seems very low priority though

**Review by Feranchz — APPROVED** [review]
I like it 🚀  Lets wait for the tests

**Pull request merged**